R/apply_frmt_methods.R

```diff
@@ -222,10 +225,10 @@ apply_frmt.frmt_combine <- function(frmt_def, .data, values, mock = FALSE, param
     mutate(
       !!values := case_when(
         .data$.is_all_missing ~ frmt_def$missing,
-        TRUE ~ str_glue(frmt_def$expression) %>% as.character()
+        TRUE ~ str_glue(!!frmt_def$expression) %>% as.character()
```

Comment:
Is there  a reason to `!!` this?

Reply:
um... for some reason this made it work. I have was having the issue when where "{`val 1} {val2}" would work but 
```{r}
frmt_def$expression <-  "{`val 1} {val2}"
str_glue(frmt_def$expression)
```
wouldn't work. So I just thought I would try the bang bang and it solved it. But not really sure why. (Yes, I realize there is like a 90% chance this will come back to bite me later